src/assets/styles/theme.scss

```diff
@@ -56,3 +56,9 @@ $BCgovInputBG: #ffffff;
 
 // Font Colors
 $BCgovFontColorInverted: #ffffff;
+
+// App Colors
+$app-blue: #1669bb; // same as the Vuetify theme primary
+$app-green: #1a9031; // same as the Vuetify theme success
+$app-red: #d3272c; // same as the Vuetify theme error
+$app-dk-blue: #38598a; // same as the custom theme dkBlue
```

Comment:
FWIW, this is still a Vuetify theme colour.

Reply:
Oh interesting, i guess i got confused because the name was custom and not an override of the actual Theme colour name, ie primary. Will update the comment!

Reply:
fixed